src/box2dx/Box2D.NetStandard/Dynamics/World/Callbacks/DebugDraw.cs

```diff
@@ -47,33 +46,26 @@ public void ClearFlags(DrawFlags flags)
         /// <summary>
         ///  Draw a closed polygon provided in CCW order.
         /// </summary>
-#pragma warning disable 618
-        [Obsolete("Look out for new calls using Vector2")]
```

Comment:
Instead of deleting, can you change the description to read "Use Vector2 overload instead" and set the Error flag to true? Then anyone already using these with Vec2 will get guidance as to how to change their code, and we can delete them at a later date.